redbot/cogs/modlog/modlog.py

```diff
@@ -26,6 +26,13 @@ def __init__(self, bot: Red):
         """Manage modlog settings."""
         pass
 
+    @checks.is_owner()
+    @modlogset.command(hidden=True, name="fixcasetypes")
+    async def reapply_audittype_migration(self, ctx: commands.Context):
+        """Command to fix misbehaving casetypes"""
+        await modlog.handle_auditype_key()
```

Comment:
ctx.typing() may be a nice addition here, as this could take a while depending on number of modlogs (FN as an example)

Reply:
Ironically, the servers which likely would want this the most can't benefit from it as discord disables typing indicators on some larger servers.